example/advection/advection_package.cpp

```diff
@@ -168,6 +169,9 @@ void PostFill(Container<Real> &rc) {
 
 // provide the routine that estimates a stable timestep for this package
 Real EstimateTimestep(Container<Real> &rc) {
+  using parthenon::X1DIR;
+  using parthenon::X2DIR;
+  using parthenon::X3DIR;
```

Comment:
Do we need this here?  Isn't this included from parthenon/package.hpp?

Reply:
yeah, added it there after I wrote this part.  I'll remove